kubernetes/spilo_kubernetes.yaml

```diff
@@ -183,13 +208,13 @@ data:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: operator
+  name: *cluster_name*
```

Comment:
There is no need to create a separate ServiceAccount/Role/RoleBinding per postgres cluster.